xml/System.Numerics/Complex.xml

```diff
@@ -1832,6 +1832,46 @@
         <altmember cref="F:System.Numerics.Complex.Zero" />
       </Docs>
     </Member>
+    <MemberGroup MemberName="op_Addition">
+      <Docs>
+        <summary>Adds a specified number to another specified number, where at least one of them is a complex number, and the other could be a double-precision real number.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The <xref:System.Numerics.Complex.op_Addition%2A> operators allow performing addition operations that involve complex numbers. They enable code such as the following:
+
+[!code-csharp[System.Numerics.Complex.Add#3](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.numerics.complex.add/cs/add3.cs#3)]
+[!code-vb[System.Numerics.Complex.Add#3](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.numerics.complex.add/vb/add3.vb#3)]
+
+The addition of a complex number, a + bi, and a second complex number, c + di, takes the following form:
+
+(a + c) + (b + d)i
+
+The addition of a real number (which can be regarded as the complex number a + 0i) and a complex number (c + di) takes the following form:
+
+(a + c) + di
+
+The addition of a complex number (a + bi) and a real number (which can be regarded as the complex number c + 0i) takes the following form:
+
+(a + c) + bi
+
+If the addition results in an overflow in either the real or imaginary component, the value of that component is either <xref:System.Double.PositiveInfinity?displayProperty=nameWithType> or <xref:System.Double.NegativeInfinity?displayProperty=nameWithType>.
```

Comment:
Since lines 1860 and 1862 apply to all three overloads, I'd keep this text here and delete it in each of the 3 overloads.